torch/_torch_docs.py

```diff
@@ -3286,6 +3286,41 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.igamma,
+           r"""
+igamma(input, other, *, out=None) -> Tensor
+
+Computes the regularized lower incomplete gamma function:
+
+.. math::
+    \text{out}_{i} = \frac{1}{\Gamma(\text{input}_i)} \int_0^{\text{other}_i} t^{\text{input}_i-1} e^{-t} dt
+
+where (1) :attr:`input` and :attr:`other` must be non-negative and (2)
+:attr:`input` and :attr:`other` must not be zero at the same index.
+Violating the constraints will return `nan`.
+
+Supports :ref:`broadcasting to a common shape <broadcasting-semantics>`
+and float inputs.
+
+.. note::
+    The backward pass with respect to :attr:`input` is not supported.
+    Please open a feature request on github if you need it.
```

Comment:
"Please open a feature request on github if you need it." -> "Please open an issue on the PyTorch Github to request it."

Reply:
Done in the recent commit